primo/opt_model/result_parser.py

```diff
@@ -102,7 +102,9 @@ def _check_column_exists(self, col_name):
         Checks if a column exists
         """
         if col_name is None:
-            raise ValueError(f"The column is not in the welldatacolumns class")
+            raise ValueError(
+                f"The column: {col_name} is not in the welldatacolumns class"
+            )

```

Comment:
Please revert it to the earlier version:
`raise ValueError("The column is not in the WellDataColumnNames object")`
The new error message reads 
`The column: None is not in the welldatacolumns class`.

Reply:
Fixed